client/transaction-pool/api/src/lib.rs

```diff
@@ -143,6 +146,8 @@ pub type TransactionFor<P> = <<P as TransactionPool>::Block as BlockT>::Extrinsi
 pub type TransactionStatusStreamFor<P> = TransactionStatusStream<TxHash<P>, BlockHash<P>>;
 /// Transaction type for a local pool.
 pub type LocalTransactionFor<P> = <<P as LocalTransactionPool>::Block as BlockT>::Extrinsic;
+/// Transaction's index within the block in which it was included.
+pub type TxIndex = usize;
```

Comment:
I wonder if we can have a specific type for `TxIndex` which serializes it as a `String` instead of having to add this `as_string` attribute on each `usize`.

It's easy to mess up...